docs/Dealing-with-SQL-files.md

```diff
@@ -1,184 +0,0 @@
-# Dealing with SQL files
```

Comment:
instead of deleting this, shall we add a link to the new page? so older links around GitHub will not be broken

Reply:
Are there any other links? If so we can just update them